File: doc-framework/test/templates/default.tpl.java
import io.kuzzle.sdk.*;
import java.util.Date;

public class CodeExampleGenericClass {
    private static Kuzzle kuzzle;

    public static void main(String[] args) {
      WebSocket ws = new WebSocket("kuzzle");
      kuzzle = new Kuzzle(ws);
      kuzzle.connect();
      [snippet-code]
    }
}

File: doc-framework/test/templates/default.tpl.go
package main

import (
	"fmt"
	"log"
	"os"
	"encoding/json"

	"github.com/kuzzleio/sdk-go/protocol/websocket"
	"github.com/kuzzleio/sdk-go/kuzzle"
	"github.com/kuzzleio/sdk-go/types"
)

func main() {
	c := websocket.NewWebSocket("kuzzle", nil)
	kuzzle, _ := kuzzle.NewKuzzle(c, nil)

	connectErr := kuzzle.Connect()
	if connectErr != nil {
		log.Fatal(connectErr)
		os.Exit(1)
	}

	[snippet-code]
}
